packages/rum-recorder/tsconfig.json

```diff
@@ -3,11 +3,7 @@
   "compilerOptions": {
     "baseUrl": ".",
     "outDir": "./bundle/",
-    "module": "es6",
-    "paths": {
-      "@datadog/browser-core": ["../core/src"],
-      "@datadog/browser-rum": ["../rum/src"]
```

Comment:
The issue could have been fixed by adding
```
"@datadog/browser-rum-core": ["../rum-core/src"]
```

Reply:
Yes, but with my proposed solution, we won't have this kind of issue anymore in the future. I think it simplifies things a little.

Reply:
agree, I think we could simplify a bit more by:
- removing rum-core tsconfig.json, since we don't build a bundle for this one (like core) 
- extract a tsconfig.bundle.json that tsconfig.json files would extend, since they are all the same now

Reply:
👌, I went one step further by replacing all `tsconfig` files used by webpack with a direct usage of `tsconfig.base.json` + a config override, it seems even simpler. I checked, we get the exact same bundles.